tests/test_models.py

```diff
@@ -37,3 +37,14 @@ def test_load_from_json(tmpdir):
         temp_json_file.write('[{"observations":[1, 2, 3]},{"observations":[4, 5, 6]}]')
     result = load_json(example_path)
     npt.assert_array_equal(result, [[1, 2, 3], [4, 5, 6]])
+
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
```

Comment:
Can some more tests be added? Useful would be:

One where the standard deviation is different to the variance (to catch the bug that s_dev is calculating the variance), ie not 0 or 1,
One with multiple patients,
One with NaNs in the input,
One with infs in the input,
One with negative numbers in the input

Reply:
Will raise as an issue for future work